### ISSUE TITLE
scRNAseq and scATACseq integration with Harmony

### ISSUE THREAD

**Issue by Aitor91** [satijalab/seurat#3370]
Hello,

I have a question regarding to the integration of my scRNAseq and scATACseq. Previously, we did the integration following the vignette that you have in your web page. It was really helpful, so thank you very much for your help and effort. 
However, there were some regions or clusters that they did not overlap very well, so I am wondering if I could use Harmony to perform an additional integration and then compare with the first one. Could be possible to do that? I have seen Harmony is used in Signac to integrate scATACseq dataset. 

Thank you again for your excellent work in advance.

Best

Aitor

**Comment by timoast** [comment]
> Could be possible to do that? I have seen Harmony is used in Signac to integrate scATACseq dataset.

I think you've answered your own question

**Comment by Aitor91** [comment]
@timoast yes as you have said I answered my own question. Ahahah. However, in Signac there is a vignette explaining who to combine 2 different scATACseq datasets, but not scRNAseq and scATACseq datasets. Anyway, I have just followed those steps and it works. Regarding to the result, in my own dataset, harmony integrates a bit better both data set. Just let you know, it could be great to add that option to your integration vignette.

Thank you very much and good job.

Aitor
